custom_components/powercalc/data/sengled/E11-N13_E11-N13A_E11-N14_E11-N14A/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "shellyplugus-d4d4da091468",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.10.0:docker"
+    },
+    "name": "Sengled Smart LED Extra Bright Soft White A19 Bulb",
+    "standby_power": 0
```

```suggestion
    "standby_power": 0.3
```

When you were unable to measure standby power because it's to low for the shelly plug to pickup you may set it to 0.3W, which will be a good estimate for Zigbee based bulbs